Cubical/Codata/M/AsLimit/Coalg.agda

```diff
@@ -1,5 +1,7 @@
-{-# OPTIONS --cubical --guardedness --safe #-}
+{-# OPTIONS --cubical --guardedness #-} --safe
```

Comment:
Why comment out the safe flag?

Reply:
I have not completed all the proofs, so I comment out safe, so I can have some postulates.